### ISSUE TITLE
Loss function of the condition U-Net in the implementation V2

### ISSUE THREAD

**Issue by asdkd** [WuJunde/MedSegDiff#89]
Thanks for your sharing about MedSegDiff V2. 

I noticed that the reported loss function of condition U-Net in the V2 paper is a combination of soft dice loss and cross-entropy loss. However, the anchor loss of condition U-Net in the V2 implementation [https://github.com/WuJunde/MedSegDiff/blob/master/guided_diffusion/gaussian_diffusion.py#:~:text=terms%5B%22loss_cal%22%5D%20%3D%20mean_flat((res%20%2D%20cal)%20**%202)](url) looks like a MSE loss instead of the Dice and BCE loss. Could you further explain a little bit about this loss function? Why is this loss inconsistent with the reported loss in the V2 paper?

**Comment by WuJunde** [comment]
I have explained it in another issue. In conclusion, mse is better, I will update arXiv
